src/main/java/net/minecraftforge/client/model/pipeline/VertexLighterFlat.java

```diff
@@ -53,8 +53,9 @@ public VertexLighterFlat(BlockColors colors)
     public void setParent(IVertexConsumer parent)
     {
         super.setParent(parent);
-        if(Objects.equal(getVertexFormat(), parent.getVertexFormat())) return;
-        setVertexFormat(getVertexFormat(parent));
+        VertexFormat format = getVertexFormat(parent);
+        if(format.equals(getVertexFormat())) return;
```

Comment:
Please humor fry and use `Objects.equal` so this can go smoothly and be accepted. ;P